src/stories/compound/TextButton.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import type { TextButtonProps } from '@components/compound/TextButton/types';
+import TextButton from '@components/compound/TextButton';
+import type { ReactElement } from 'react';
+
+export default {
+  title: 'Component/TextButton',
```

Comment:
```suggestion
  title: 'Component/Compound/TextButton',
```
로 나중에 변경해 주시면 좋을 것 같아요^^